tests/snippets/list.py

```diff
@@ -488,3 +488,27 @@ def bad_iter_assign():
 assert x == [0, 1, 2, 3, 4, 5]
 x = list(range(10))
 del x[-5:-1:-1]
+
+# list gt, ge, lt, le
+assert_raises(TypeError, lambda: [0, []] < [0, 0])
+assert_raises(TypeError, lambda: [0, []] <= [0, 0])
+assert_raises(TypeError, lambda: [0, []] > [0, 0])
+assert_raises(TypeError, lambda: [0, []] >= [0, 0])
+
+assert_raises(TypeError, lambda: [0, 0] < [0, []])
+assert_raises(TypeError, lambda: [0, 0] <= [0, []])
+assert_raises(TypeError, lambda: [0, 0] > [0, []])
+assert_raises(TypeError, lambda: [0, 0] >= [0, []])
+
+assert [0, 0] < [1, -1]
+assert [0, 0] < [0, 0, 1]
+assert [0, 0] < [0, 0, -1]
+assert [0, 0] <= [0, 0, -1]
+
+assert [0, 0] > [-1, 1]
+assert [0, 0] >= [-1, 1]
+assert [0, 0, 0] >= [-1, 1]
+
+assert [0, 0] <= [0, 1]
+assert [0, 0] <= [0, 0]
+assert [0, 0] >= [0, 0]
```

Comment:
Adding tests with inf, nan will be good
```
assert [nan] == [nan]
assert [nan] >= [nan]
...
```

Reply:
I added a test case and modified the `seq_ge` and` seq_le` codes.
Can I get a review again?